ros_ws/src/simulation/simulation_tools/src/simulation_tools/lap_timer.py

```diff
@@ -34,6 +35,8 @@ def __init__(self, name, checkpoints):
         self.next_checkpoint = 0
         self.history = []
         self.start = None
+        self.msg_seq = 0
+        self.pub = rospy.Publisher("/lap_time", DurationStamped, queue_size=1)
```

Comment:
Why do you use a DurationStamped instead of `std_msgs/Duration Message`?
Does `marti_common_messages` need to be installed manually?
The variable names should not be abbreviations, please use something like `message_index` and `lap_time_publisher`.

Reply:
I figured the increasing sequence numbers might be useful for some applications down the line and the overhead doesn't seem that big to me. Unfortunately, `marti_common_messages` would need to be installed manually. @Sami19944 , would such a message header be useful for you?

Reply:
i cant think of a scenario where i can use them.  important are the lap times at the end of a finished lap